axis.go

```diff
@@ -145,6 +145,19 @@ func (a *Axis) sanitizeRange() {
 		a.Min--
 		a.Max++
 	}
+
+	var (
+		min  = math.Float64bits(a.Min)
+		max  = math.Float64bits(a.Max)
+		nULP = max - min
+	)
+	const minULP = 20 // arbitrary value found experimentally with gonum/plot#514.
+	// add enough ULP space between min and max
+	// see https://github.com/gonum/plot/issues/514
+	if nULP <= minULP {
+		a.Min = math.Float64frombits(min - minULP/2)
```

Comment:
Not that this is incorrect, but this gives a range that is higher than minULP for all cases. Is that what you want?

Reply:
It's an oversight.
Thanks for spotting this.

Are you onboard with the general direction of this PR?
And the choice of handling this issue at the level of the Axis instead of at the level of the Talbot algorithm?

Reply:
Yes, it seems reasonable to me.